gempak/Dockerfile

```diff
@@ -1,27 +1,55 @@
+#####
+# Dockerfile configuration for Unidata
+# Gempak Docker image.
+#####
+
 FROM centos
 
+###
+# Update package manager, install a few things.
+###
 RUN yum update yum
 
 RUN yum install -y wget csh openmotif openmotif-devel gcc gcc-c++ gcc-gfortran \
 	zlib-devel bzip2 libX11-devel libXt-devel libXext libXp-devel \
 	libXft-devel libXtst-devel xorg-x11-xbitmaps flex byacc -y
 
+###
 # Set up gempak directory
-
+###
 RUN mkdir /home/gempak
-
 ENV HOME /home/gempak
-
 WORKDIR /home/gempak
 
-# Output directory
-
+###
+# Set up the output directory
+###
 RUN mkdir /tmp/output
 
-# Install GEMPAK
+###
+# Install GEMPAK,
+# set up some stuff in bashrc.
+###
 
 COPY install_gempak.sh /home/gempak/
-COPY Dockerfile /home/gempak/
-RUN /home/gempak/install_gempak.sh 
+RUN /home/gempak/install_gempak.sh
 RUN echo '. /home/gempak/NAWIPS/Gemenviron.profile' >> ~/.bashrc
-CMD bash
+
+###
+# Copy some other files over.
+###
+COPY Dockerfile /home/gempak/
```

Comment:
So that it is handily available to anybody who is curious but may not want to grab it from git. Basically a bit of convenience that costs nothing.

On Tue, Oct 20, 2015 at 4:56 PM, Ryan May <notifications@github.com>
wrote:

>>  RUN echo '. /home/gempak/NAWIPS/Gemenviron.profile' >> ~/.bashrc
>> -CMD bash
>> +
>> +###
>> +# Copy some other files over.
>> +###
>> +COPY Dockerfile /home/gempak/
> I'm curious: Why copy the Dockerfile *inside* the container?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Unidata/Unidata-Dockerfiles/pull/7/files#r42565686